src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts

```diff
@@ -1138,6 +1138,11 @@ class normalCrudActions implements editDocumentCrudActions {
     revisionsCount = ko.observable<number>();
     timeSeriesCount: KnockoutComputed<number>;
 
+    attachmentsCountBadge: KnockoutComputed<string>;
```

Comment:
can we create single formatting function instead of creating many new funcions?

Reply:
done - I forced push

Reply:
let's use this function directly in view. Like I said during our call. We don't have to have so many fields in this class. Let's keep it clean.

Reply:
in html:

data-bind="text: $root.formatBadge(attachmentsCount)"

Reply:
ok - forced pushed again